owncloud/ios-app
# Collaboration/transifex

### Description of the Pull Request


This pull request add the posibility to use Transifex platform to add the translations automatically to the project

### Related issue

https://github.com/owncloud/ios-app/issues/45

### How can be tested

Once this is on Master the transifex bot should can start to update the project with the new strings files
